python/factorial.py

```diff
@@ -0,0 +1,8 @@
+def factorial(n):
+    if n == 0:
```

Comment:
Please handle negative cases.

Reply:
are you sure !!!!! factorial does not exist for negative numbers

Reply:
Exactly!
Just print a statement saying that or raise an exception. :)